creator/status/banners/queries.py

```diff
@@ -0,0 +1,35 @@
+from graphene import relay
+from graphene_django.filter import DjangoFilterConnectionField
+from graphql import GraphQLError
+from django_filters import FilterSet, OrderingFilter
+
+from creator.status.banners.nodes import BannerNode
+from creator.status.banners.models import Banner
+
+
+class BannerFilter(FilterSet):
+    order_by = OrderingFilter(fields=("created_on",))
+
+    class Meta:
+        model = Banner
+        fields = ["enabled"]
+
+
+class Query(object):
+    banner = relay.Node.Field(BannerNode, description="Get a single banner")
+    banners = DjangoFilterConnectionField(
```

Comment:
Can you rename this `allBanners` to be conformant with all our other list queries?

Reply:
@dankolbman So I actually had it as `allBanners` before bc thats what the `startresource` cmd generated for me but I changed it because the other things within the Status query don't follow that naming convention (`settings`, `features`). Should I still change it to `allBanners`?